test/vanilla/ignite-new.test.ts

```diff
@@ -1,70 +1,349 @@
 import { filesystem } from "gluegun"
 import * as tempy from "tempy"
-import { runIgnite, runError, testSpunUpApp } from "../_test-helpers"
+import { runIgnite, runError, run } from "../_test-helpers"
 
 const APP_NAME = "Foo"
-
 const originalDir = process.cwd()
-let tempDir: string
 
-beforeEach(() => {
-  tempDir = tempy.directory({ prefix: "ignite-" })
-})
+describe("ignite new", () => {
+  describe("errors", () => {
+    let tempDir: string
 
-afterEach(() => {
-  filesystem.remove(tempDir) // clean up our mess
-})
+    beforeEach(() => {
+      tempDir = tempy.directory({ prefix: "ignite-" })
+    })
 
-describe("Checking for ignite. 🪔", () => {
-  test(`ignite new (invalid bundle ID)`, async () => {
-    const result = await runError(`new BadBundleID --bundle thisisbad --yes`)
-    expect((result as any).stdout).toContain(`Invalid Bundle Identifier.`)
+    afterEach(() => {
+      filesystem.remove(tempDir) // clean up our mess
+    })
+
+    test(`invalid bundle id "thisisbad" throws expected error`, async () => {
+      const result = await runError(`new BadBundleID --bundle thisisbad --yes`)
+      expect((result as any).stdout).toContain(`Invalid Bundle Identifier.`)
+    })
   })
-})
 
-describe("Igniting new app! 🔥\nGo get a coffee or something. This is gonna take a while.", () => {
-  test(`ignite new ${APP_NAME}`, async () => {
-    const result = await runIgnite(`new ${APP_NAME} --debug --packager=npm --yes`, {
-      pre: `cd ${tempDir}`,
-      post: `cd ${originalDir}`,
+  describe(`ignite new ${APP_NAME} --debug --packager=npm --yes`, () => {
+    let tempDir: string
+    let result: string
+    let appPath: string
+
+    beforeAll(async () => {
+      tempDir = tempy.directory({ prefix: "ignite-" })
+      result = await runIgnite(`new ${APP_NAME} --debug --packager=npm --yes`, {
+        pre: `cd ${tempDir}`,
+        post: `cd ${originalDir}`,
+      })
+      appPath = filesystem.path(tempDir, APP_NAME)
+    })
+
+    afterAll(() => {
+      filesystem.remove(tempDir) // clean up our mess
+    })
```

Comment:
Now this test suite will set everything else up once, then we can create tests to describe individual regions we are testing for. That way jest will report when one piece fails, but not fail the entire test block

Also it will run these tests in parallel, which offers a modest speed increase